packages/lesswrong/components/users/NewUserCompleteProfile.tsx

```diff
@@ -141,88 +155,103 @@ const NewUserCompleteProfile: React.FC<NewUserCompleteProfileProps> = ({ current
-          Get weekly emails with selected posts
-        </Typography>
-        <Typography variant='body1' className={classes.sectionHelperText} gutterBottom>
-          The EA Forum Digest is curated by the Forum team, and features highlights from
-          every week, announcements, and more.{" "}
-          <Link to={eaForumDigestLink} target="_blank" rel="noreferrer">
-            See recent issues here
-          </Link>.
-        </Typography>
-        <FormControlLabel
-          control={
-            <Checkbox
-              checked={subscribeToDigest}
-              onChange={event => setSubscribeToDigest(event.target.checked)}
-            />
-          }
-          label='Yes, subscribe me to EA Forum digest emails'
-        />
-      </div>}
-      {/* TODO: Something about bio? */}
-      <div className={classes.submitButtonSection}>
-        {isEAForum &&
-          <Typography variant="body1" className={classnames(classes.sectionHelperText, classes.tosText)} gutterBottom>
-            I agree to the <TosLink />, including my content being available
-            under a <LicenseLink /> license.
+        {isEAForum && <div className={classes.section}>
+          <Typography variant='display1' className={classes.sectionHeadingText} gutterBottom>
+            Get weekly emails with selected posts
           </Typography>
-        }
-        <EAButton onClick={handleSave} disabled={!!validationError}>
-          Submit
-        </EAButton>
+          <Typography variant='body1' className={classes.sectionHelperText} gutterBottom>
+            The EA Forum Digest is curated by the Forum team, and features highlights from
+            every week, announcements, and more.{" "}
+            <Link to={eaForumDigestLink} target="_blank" rel="noreferrer">
+              See recent issues here
+            </Link>.
+          </Typography>
+          <FormControlLabel
+            control={
+              <Checkbox
+                checked={subscribeToDigest}
+                onChange={event => setSubscribeToDigest(event.target.checked)}
+              />
+            }
+            label='Yes, subscribe me to EA Forum digest emails'
+          />
+        </div>}
+        {isEAForum && <div className={classes.section}>
+          <div className={classes.callout}>
+            <h2 className={classes.calloutHeadingText}>
+              Chat with an EA Forum Team member
+            </h2>
+            <Typography variant='body1' className={classes.sectionHelperText}>
+              We want to hear from you! Sign up for a user interview with a member of the EA Forum Team
+              and we'll answer any questions you have about EA or the Forum.{" "}
+              <Link to="https://savvycal.com/cea/forum-team" target="_blank" rel="noreferrer">
+                Book a call here
+              </Link>.
+            </Typography>
+          </div>
+        </div>}
```

Comment:
@oetherington The only new thing here (other than the analytics context) is this section